nss-day-cohort-60/gamer-rater-client-v1
# Removed references to Level Up and added email field for registration

# Description

Updated the `Register.js` component to remove reference to Level Up and added a new text field to capture email address. New reference added and key added to object that is stringified for request body.

## Steps to tests

1. Start client with `npm start`
2. Click on **Register** in navigation bar
3. Fill out all fields
4. Make sure your API is updated with authentication code and started
5. Click **Register** button and verify you get a token in the response body
